regions/io/core.py

```diff
@@ -3,19 +3,56 @@
 from __future__ import absolute_import, division, print_function
 from collections import OrderedDict
```

Comment:
collections.OrderedDict has been replaced by RegionVisual and this import can be removed